bin/travis-ci/conf.vexpress_ca15_tc2_qemu

```diff
@@ -21,7 +21,7 @@
 console_impl=qemu
 qemu_machine="vexpress-a15"
 qemu_binary="qemu-system-arm"
-qemu_extra_args="-nographic -m 1G -tftp /tftpboot"
+qemu_extra_args="-nographic -m 1G -tftp ${UBOOT_TRAVIS_BUILD_DIR}"
```

Comment:
The x86 script has "-netdev user,id=net0" in qemu_extra_args. Is something similar not needed for the vexpress scripts too, or do they default to having an emulated NIC?